.golangci.yml

```diff
@@ -45,6 +45,9 @@ linters:
     - gochecknoglobals
     - gochecknoinits
     - dupl
+    - gomnd
```

Comment:
Why are these being disabled?

Reply:
as you can see I updated version of golangci-lint, looks like some new linters were enabled by default there. They produced many warnings that look not very sensible - like "magic number" arguments, too long function.
I tried to follow overall style of existing code as much as possible, so I thought that it makes no sense to refactor according to that new warnings and just disabled them.
Anyway, if at some moment someone wants to enable them, it's better to refactor the whole project, and obviously it will be worth separate PR